cli/flags.rs

```diff
@@ -149,157 +86,236 @@ To evaluate code from the command line:
+      .min_values(0)
+      .takes_value(true)
+      .use_delimiter(true)
+      .require_equals(true)
+      .help("Allow network access"),
+    Arg::with_name("allow-env")
+      .long("allow-env")
+      .help("Allow environment access"),
+    Arg::with_name("allow-run")
+      .long("allow-run")
+      .help("Allow running subprocesses"),
+    Arg::with_name("allow-hrtime")
+      .long("allow-hrtime")
+      .help("Allow high resolution time measurement"),
+    Arg::with_name("allow-all")
+      .short("A")
+      .long("allow-all")
+      .help("Allow all permissions"),
+  ]
+}
+
+fn configuration_args<'a, 'b>() -> Vec<Arg<'a, 'b>> {
+  vec![
+    Arg::with_name("config")
+      .short("c")
+      .long("config")
+      .value_name("FILE")
+      .help("Load tsconfig.json configuration file")
+      .takes_value(true),
+    Arg::with_name("importmap")
+      .long("importmap")
+      .value_name("FILE")
+      .help("Load import map file")
+      .long_help(
+        "Load import map file
 Specification: https://wicg.github.io/import-maps/
 Examples: https://github.com/WICG/import-maps#the-import-map",
-        )
-        .takes_value(true)
-        .global(true),
-    ).arg(
-      Arg::with_name("seed")
-        .long("seed")
-        .value_name("NUMBER")
-        .help("Seed Math.random()")
-        .takes_value(true)
-        .validator(|val: String| {
-          match val.parse::<u64>() {
-            Ok(_) => Ok(()),
-            Err(_) => Err("Seed should be a number".to_string())
-          }
-        })
-        .global(true),
-    ).arg(
-      Arg::with_name("lock")
-        .long("lock")
-        .value_name("FILE")
-        .help("Check the specified lock file")
-        .takes_value(true)
-        .global(true),
-    ).arg(
-      Arg::with_name("lock-write")
-        .long("lock-write")
-        .help("Write lock file. Use with --lock.")
-        .global(true),
-    ).arg(
-      Arg::with_name("v8-options")
-        .long("v8-options")
-        .help("Print V8 command line options")
-        .global(true),
-    ).arg(
-      Arg::with_name("v8-flags")
-        .long("v8-flags")
-        .takes_value(true)
-        .use_delimiter(true)
-        .require_equals(true)
-        .help("Set V8 command line options")
-        .global(true),
-    ).subcommand(
-      SubCommand::with_name("version")
-        .about("Print the version")
-        .long_about("Print current version of Deno.
-
-Includes versions of Deno, V8 JavaScript Engine, and the TypeScript
-compiler.",
-        ),
-    ).subcommand(
-      SubCommand::with_name("bundle")
-        .about("Bundle module and dependencies into single file")
-        .long_about(
-          "Output a single JavaScript file with all dependencies.
+      )
+      .takes_value(true),
+  ]
+}
 
-If a out_file argument is omitted, the output of the bundle will be sent to
-standard out.
+fn runtime_args<'a, 'b>() -> Vec<Arg<'a, 'b>> {
+  vec![
+    Arg::with_name("no-fetch")
```

Comment:
This is not a runtime arg, it should affect `deno fetch`.

IMO there should be a collection for `--config`, `--import-map`, `--no-fetch` and `--lock` that applies to anything which `deno fetch`es user code.

Reply:
I don't really see a benefit of adding `--no-fetch` to `deno fetch` - it downloads and compiles code, so it'd be pretty strange to prohibit it from downloading modules...

Reply:
`--no-fetch` prohibits downloading modules so it should apply to anything that might download user modules.

It would certainly be strange if it was (explicitly) applied to a subcommand which _only_ downloads modules but as you said it compiles, too. That makes it a meaningful option.